.github/pull_request_template.md

```diff
@@ -1,21 +1,38 @@
+<!-- Use the first line of the commit description as the title for the PR -->
+
 ## Description
```

Comment:
For my personal taste this is still too noisy. I prefer a minimal version, but I can live with something like this...

Reply:
Would you delete the commented out instructions or the still visible part of the template. I've reduced the verbosity as much as possible but would like to still give some useful information in the template to try to enforce useful and uniform commit descriptions.

If we simplify it even more then I guess the actual question is if we want a PR template or not. I'm fully in favor of using a template, but if most of developers think the template is just noise, I'll remove it.